bin/tass

```diff
@@ -78,6 +78,19 @@ when 'create'
         end
       end
     end
+
+    if config.key?(:asg_metrics)
```

Comment:
We should not this boolean option in docs/CONFIG_OPTIONS.md

Reply:
Fixed